README.md

```diff
@@ -31,7 +31,7 @@
 
 ## 💡 Motivation <a id="motivation" />
 
-I once came across the fact that none of the Golang cache libraries are truly contention-free. All of them are just a standard map with mutex and some eviction policy. Unfortunately, these are not able to reach the speed of caches in other languages (such as [Caffeine](https://github.com/ben-manes/caffeine)). For example, the fastest cache from Dgraph labs called [Ristretto](https://github.com/dgraph-io/ristretto), which was faster than competitors by 30% at best (Otter is many times faster) and had a [disgusting hit ratio](https://github.com/dgraph-io/ristretto/issues/336) even though README says otherwise. This can be a problem in different applications because no one wants to bump the performance of a cache library and its bad hit ratio 🙂. As a result, I wanted to get the fastest, easiest-to-use cache with excellent hit ratio and support from the authors and Otter is designed to correct this unfortunate misunderstanding.
+I once came across the fact that none of the Go cache libraries are truly contention-free. Most of them are a map with a mutex and an eviction policy. Unfortunately, these are not able to reach the speed of caches in other languages (such as [Caffeine](https://github.com/ben-manes/caffeine)). For example, the fastest cache from Dgraph labs called [Ristretto](https://github.com/dgraph-io/ristretto), was faster than competitors by 30% at best (Otter is many times faster) but had [poor hit ratio](https://github.com/dgraph-io/ristretto/issues/336), even though its README says otherwise. This can be a problem in real-world applications, because no one wants to bump into performance of a cache library 🙂. As a result, I wanted to make the fastest, easiest-to-use cache with excellent hit ratio.
```

Comment:
> [..] and support from the authors and Otter is designed to correct this unfortunate misunderstanding.

I tried to keep the original tone and style, but I've removed this part, because I don't understand what the phrase refers to, in the context of the section. I believe it might have been a typo.

Reply:
Oh, I wrote the README in a hurry, so there might be typos in it. The motivation could be rewritten, because its style is really not very good. But its style reflects the main reason why otter was created in the first place, namely the very small hit ratio of ristretto, which doesn't match the README and the overall quality of in-memory cache libraries in go. I was just really frustrated.

Reply:
That turned out really well. I'll probably rewrite this section sometime or even delete it, but for now it looks like a great improvement.